pkg/sfu/streamallocator.go

```diff
@@ -1030,6 +1127,24 @@ func (s *StreamAllocator) maybeGratuitousProbe() {
 	s.prober.AddCluster(int(committedChannelCapacity+probeRateBps), int(expectedRateBps), GratuitousProbeMinDurationMs, GratuitousProbeMaxDurationMs)
 
 	s.setState(State_GRATUITOUS_PROBING)
+	return true
+}
+
+//------------------------------------------------
+
+func appendTrack(t *Track, m *map[string][]string) {
+	if *m == nil {
+		*m = make(map[string][]string)
+	}
+
+	peerID := t.PeerID()
+	trackID := t.ID()
+	peer, ok := (*m)[peerID]
+	if !ok {
+		peer = make([]string, 2)
```

Comment:
doing the above will create a slice in the end with [nil, nil, trackID]

```suggestion
		peer = make([]string, 0, 2)
```